### PR TITLE
Feature/tenant pinboard#13

### PR DESCRIPTION
**Pull request by Gubbsy**
Implemented pinboard

### CODE REVIEW

**Review by tobybessant — APPROVED** [review]
Looks good

File: src/app/models/response/tenant/pinboard-response.interface.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default interface IPinboardResponse {
+    text: string;
```
Comment (tobysmith568) [inline src/app/models/response/tenant/pinboard-response.interface.ts]: 4 char indent

File: src/app/services/api/tenant/pinboard-service.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+@Injectable()
+export default class TenantPinBoardService {
+
+  constructor(private apiService: TenantAPIService) {
+  
+  }
+
+  async getPinBoard(): Promise<IAPIResponse<IPinboardResponse>>  {
+      return await this.apiService.Post<IPinboardResponse>("getPinBoard", null);
```
Comment (tobysmith568) [inline src/app/services/api/tenant/pinboard-service.ts]: 4 char indent

File: src/app/views/tenant/pinboard/pinboard.module.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { FormsModule } from "@angular/forms";
+import { Routes, RouterModule } from "@angular/router";
+import { IonicModule } from "@ionic/angular";
+import { PinboardPage } from "./pinboard.page";
+import TenantPinBoardService from 'src/app/services/api/tenant/pinboard-service';
```
Comment (tobysmith568) [inline src/app/views/tenant/pinboard/pinboard.module.ts]: Single quotes

**Review by tobysmith568 — CHANGES_REQUESTED** [review]

**Review by tobysmith568 — APPROVED** [review]

**Pull request merged**